src/ucp/core/ucp_context.c

```diff
@@ -136,6 +136,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "the eager_zcopy protocol",
    ucs_offsetof(ucp_config_t, ctx.rndv_perf_diff), UCS_CONFIG_TYPE_DOUBLE},
 
+  {"RNDV_MAX_DIFF", "10",
+   "How many times the RNDV lane can be worse than the best RNDV lane to use "
```

Comment:
better split to two lines.
Also maybe rephrase, like: "Maximum allowed difference in rendezvous lane performance"

Reply:
done